pkgs/by-name/mi/microsocks/package.nix

```
@@ -0,0 +1,29 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib
+}:
+stdenv.mkDerivation rec {
+  pname = "microsocks";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "rofl0r";
+    repo = "microsocks";
+    rev = "v${version}";
+    fetchSubmodules = true;
```

```suggestion
    fetchSubmodules = true;
```
What happens if you remove this?